packages/ndla-howto/package-lock.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Det er en fordel om alle bruker yarn og ikkje npm for då unngår vi å ha både package-lock.json og yarn.lock.